pkg/controller/gc_controller.go

```diff
@@ -84,9 +86,63 @@ func NewGCController(
 		restoreClient:       restoreClient,
 		logger:              logger,
 	}
-}
 
-var _ Interface = &gcController{}
+	backupInformer.Informer().AddEventHandler(
+		cache.ResourceEventHandlerFuncs{
+			UpdateFunc: func(old interface{}, new interface{}) {
```

Comment:
Since `new` is a reserved keyword, it might be better not to use it. Looking at the kube source, I've seen
- `func(oldObj, newObj interface{})`
- `func(old, cur interface{})`

I think I prefer the first form.

Reply:
Also since you're not using `old`, you can just do `func(_, newObj interface{})`